### ISSUE TITLE
Add container's local IP into the /status output 

### ISSUE THREAD

**Issue by OlegKuzovkov** [aerokube/selenoid#316]
Hi guys!

Could you please add container's local IP attached to the driver's session into the output of the /status endpoint?

This might be useful for someone who want to be able to send some requests directly into container for different purposes(video recording on demand, by sending request to turn it on/off and pull the file as example). 

Right now every driver has a unique session_id, that is displayed in the /status. By this id we can see the docker container id that was created for the session:
![screenshot at dec 21 14-57-01](https://user-images.githubusercontent.com/9959366/34272889-19ef5a44-e660-11e7-9c7f-ab27ec6790c5.png)

To get the local Ip of a container you can use "docker inspect <container_id>", where container_id will be taken from the /status above.
![screenshot at dec 21 14-58-45](https://user-images.githubusercontent.com/9959366/34272917-36e50dc4-e660-11e7-9b83-2645ea0237e0.png)

It would be super cool if you add this ip_address field into the /status output!

Thanks in advance!

**Comment by vania-pooh** [comment]
Is now available in `aerokube/selenoid:latest` container. Will be released in 1.4.2.

**Comment by vania-pooh** [comment]
Released in `1.4.2`.
@addrians status API format has changed in new release. Now we return something like this:
```
{
  "total": 5,
  "used": 1,
  "queued": 0,
  "pending": 0,
  "browsers": {
    "firefox": {
      "55.0": {
        "unknown": {
          "count": 1,
          "sessions": [
            {
              "id": "c9ed6055-e7c2-46e9-98de-4241a25ad420",
              //Note this change!!!
              "container": {
                "id": "ec8338eb8f271bc91dae7eee0cc16924e5e78fb7cd1b40ad59d50a99fc7534a1",
                "ip": "172.17.0.3"
              },
              "vnc": false,
              "screen": "1920x1080x24",
              "caps": {
                //...
              }
            }
          ]
        }
      }
    }
  }
}
```
